cdap-proto/src/main/java/co/cask/cdap/proto/security/SecureStoreEntry.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.security;
+
+import java.util.Objects;
+
+/**
+ * Represents an entry in the listing of secure store elements.
+ */
+public class SecureStoreEntry {
+
+  private final String name;
```

Comment:
so the entry has just name and description? how about values?

Also, again since this class is user facing, I'd prefer not to expose "Store" to users. It will be simpler if users only know about Secure Keys right? I could be wrong though.